weave/flow/eval.py

```diff
@@ -49,7 +49,7 @@ def async_call(
     return asyncio.to_thread(func, *args, **kwargs)
 
 
-class EvaluationResults(weave.Object):
+class EvaluationResults(Object):
```

Comment:
i think it is cleaner to use `weave.Object` in most cases

Reply:
It causes an import cycle in this case which is why it's changed

Reply:
oh, it is because of the `__init__` problems i called out below

Reply:
They are changed at the `__init__` level -- you can try adding it back, you'll get a cycle!